lib/PermissionedRoute/NoPermissionsMessage/NoPermissionsMessage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { MessageBanner } from '@folio/stripes/components';
+
+const NoPermissionsMessage = ({ returnLinkLabelId, returnLink }) => (
+  <MessageBanner type="warning">
+    <FormattedMessage id="stripes-acq-components.noPermissionsMessage" />
+    <Link to={returnLink}>
+      <FormattedMessage id={returnLinkLabelId} />
+    </Link>
```

Comment:
Do we need explicit module link to go back to in case of no permission?
Maybe something like `<button onClick={history.goBack}>` will do?

Reply:
`history.goBack` won't work in some cases if the user uses a direct link. We want to provide a root module link